src/components/Pages/Problem/Index.tsx

```
@@ -82,10 +82,10 @@ export default function Problem() {
   const containerRef = useRef<HTMLDivElement>(null);
   const navigate = useNavigate();
   const { isFullScreenEnabled, toggleFullScreen } = useFullScreen();
-  const { postMessage } = useBroadcastChannel<{ type: string; submissions: problemsubmission[],userId: string }>(
+  const { postMessage } = useBroadcastChannel<{ type: string; submissions: problemsubmission[], userId: string }>(
     'submission-sync',
     (data) => {
-     if (data.type !== 'submission') return;
+      if (data.type !== 'submission') return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)